Makefile

```diff
@@ -0,0 +1,50 @@
+############################################################################
+# This file is part of the Symfony CMF package.                            #
+#                                                                          #
+# (c) 2011-2017 Symfony CMF                                                #
+#                                                                          #
+# For the full copyright and license information, please view the LICENSE  #
+# file that was distributed with this source code.                         #
+############################################################################
+
+CONSOLE=vendor/symfony-cmf/testing/bin/console
+
+list:
+	@echo 'test:                  will run all tests'
+	@echo 'test_unit:             will run unit tests only'
+	@echo 'test_integration_phpcr:  will run integration tests on PHPCR'
+	@echo 'test_integration_orm:  will run integration tests on ORM'
+
+test_integration_phpcr:
+	@echo
+	@echo '+++ create PHPCR +++'
+	${CONSOLE} doctrine:phpcr:init:dbal --drop --force
+	${CONSOLE} doctrine:phpcr:repository:init
+	@echo '+++ run PHPCR integratin tests +++'
+	vendor/bin/simple-phpunit --testsuite "phpcr"
+	${CONSOLE} doctrine:database:drop --force
+
+test_integration_orm:
+	@echo
+	@echo '+++ Create ORM database +++'
+	${CONSOLE} doctrine:schema:drop --env=orm --force
+	${CONSOLE} doctrine:database:create --env=orm
+	${CONSOLE} doctrine:schema:create --env=orm
+	@echo '+++ run PHPCR integratin tests +++'
+	vendor/bin/simple-phpunit --testsuite "orm"
+	${CONSOLE} doctrine:database:drop --force
+
+
+test_unit:
+	@echo
+	@echo '+++ run unit tests +++'
+	vendor/bin/simple-phpunit --testsuite "unit tests"
+
+test_installation:
```

Comment:
@dbu this would be a idea to test the installation on the branch itself.

Reply:
that looks quite cool!

i wonder if we can put most of whats in this makefile into shell scripts in symfony-cmf/testing. the orm/phpcr flag could be a parameter to the script.

i would take the installation test out of the default set of tests. locally it won't work unless the commit has been pushed, and we don't need that in each combination of the matrix. we could have just one test case defined in the matrix in .travis.yml that runs this instead of the normal test suite.

Reply:
Indeed, i will set an own matrix entry for that one. 

yes we should exclude the scripts into own scripts if they get longer than 3-4 lines. Same for travis. I forked sonata's `dev-kit` yesterday and will put that stuff into that application. So we can serve all packages with Makefiles and travis files. this one should be a blueprint to generalize it next week, when we have got Mayflower Barcamp :-)

Reply:
why not put the scripts into symfony-cmf/testing? they are about testing, after all... afaik the dev-kit is a way to keep boilerplate files of each repo in sync. that is the next best option, but i think using symfony-cmf/testing would be simpler and work just as well.

Reply:
Sure. But i can do one step after each only. So:

1. Getting this working in complete
2. Move tasks to scripts in `testing`
3. Serve Makefile and .travis.yaml from a dev-kit, calling scripts in `testing` to all of our bundles?
4. Fix broken stuff
5. Happy

I would even move the `testing-install` stuff into `testing` later. Currently it is the easiest way to get an overview an serve changes.

Reply:
sounds like a great plan!